### PR TITLE
Update Accounts Rewards Parameters

### PR DESCRIPTION
**Pull request by amirhossein-ceh**
add missing parameter ( bucket ) to hotspots rewards

### CODE REVIEW

**Comment by PharkMillups** [comment]
Thanks! @madninja Can you take a quick look please?

**Comment by amirhossein-ceh** [comment]
@madninja 
@PharkMillups 
@abhay 
@jamiew 
Hi guys , could you please take a quick look here  ?
actually it's a missing parameter and it's urgent to solve this problem ASAP!

File: docs/api/blockchain/hotspots.mdx [hunk 1]
```diff
@@ -2092,6 +2092,7 @@ _Query Parameters_
 | ------------------- | -------- | -------------------------------------- |
 | max_time (required) | _string_ | Last timestamp to include rewards for  |
 | min_time (required) | _string_ | First timestamp to include rewards for |
+| bucket (optional) | _string_ | Get Data in Special Bucket ( hour / month / week ) |
```
Comment (madninja) [inline docs/api/blockchain/hotspots.mdx]:
```suggestion
| bucket (optional) | _string_ | Get data in buckets ( hour / month / week ) |
```

**Comment by PharkMillups** [comment]
@amirhossein-ceh Sorry for the delay here. Once small change suggested per above. After that, we can get it merged. Thanks!

**Review by cokes518 — APPROVED** [review]

**Pull request merged**